hw/top_darjeeling/data/top_darjeeling.hjson

```diff
@@ -73,7 +73,6 @@
     // src:  From which clock source is the clock derived
     // div:  Ratio between derived clock and source clock
     derived_srcs: [
-      { name: "io_div2", aon: "no", div: 2, src: "io", freq: "500000000" }
```

Comment:
`io_div2` is still mentioned in Darjeeling's testplans, some DV code, and configuration files. I don't think this is a critical blocker to this PR, but I suggest we create an issue to track the removal / cleanup of those stale references.

Reply:
I removed most references to that clock.